features/step_definitions/cost_steps.rb

```diff
@@ -0,0 +1,16 @@
+Given(/^I have ordered "(.*?)" "(.*?)"$/) do |num, item|
+  order = num.to_i.times{Order.new(item.to_s)}.to_a
```

Comment:
Useless assignment to variable - `order`.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.